jabgui/src/main/java/org/jabref/gui/preview/PreviewViewer.java

```diff
@@ -222,17 +228,73 @@ private String formatError(BibEntry entry, Throwable exception) {
     }
 
     private void setPreviewText(String text) {
+        String coverIfAny = "";
+        Optional<String> image = getCoverImageURI();
+        if (image.isPresent()) {
+            coverIfAny = "<img style=\"border-width:1px; border-style:solid; border-color:black; display:block; height:12rem;\" src=\"%s\"> <br>".formatted(image.get());
+        }
+
         layoutText = """
-                <html>
-                    <body id="previewBody">
-                        <div id="content"> %s </div>
-                    </body>
-                </html>
-                """.formatted(text);
+                    <html>
+                        <body id="previewBody">
+                            %s <div id="content"> %s </div>
+                        </body>
+                    </html>
+                """.formatted(coverIfAny, text);
         highlightLayoutText();
         setHvalue(0);
     }
 
+    private Optional<String> getCoverImageURI() {
```

Comment:
This logic does not make use of `isbn-xyz` filenames discussed at `org.jabref.gui.importer.BookCoverFetcher#withAttachedCoverFileIfExists`

Use case: I have a directory with covers of my books - and not attached to the bib entry. They should be displayed, too -- without explicit change of the bib entry.